metadata.json

```diff
@@ -81,8 +81,8 @@
       "version_requirement": ">= 2.0.0 < 4.0.0"
     },
     {
-      "name": "stahnma/epel",
-      "version_requirement": ">= 1.0.0 < 2.0.0"
+      "name": "puppet/epel",
+      "version_requirement": ">= 1.0.0 < 4.0.0"
```

Comment:
The minimum version should be `3.0.0`.  There are no earlier versions in the `puppet` namespace.

Reply:
Earlier today, I released `puppet/epel` `3.0.1`, which should fix the build failures.  Could you actually set that as the minimum version?